uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveHoursBalanceChangeCalculationTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_BAO_LeaveRequest as LeaveRequest;
+use CRM_HRLeaveAndAbsences_Service_LeaveHoursBalanceChangeCalculation as LeaveHoursBalanceChangeCalculation;
+
+/**
+ * @group headless
+ */
+class CRM_HRLeaveAndAbsences_Service_LeaveHoursBalanceChangeCalculationTest extends BaseHeadlessTest {
+
+  use CRM_HRLeaveAndAbsences_LeaveRequestHelpersTrait;
+
+  public function testGetAmountReturnsCorrectlyForFromDate() {
+    $leaveDaysCalculationService = new leaveHoursBalanceChangeCalculation();
```

Comment:
s/new leave/new Leave/ 
Also, since you're instantiating this same class in all the tests, why not doing this in the setUp method?